lua/nvim-ts-autotag/internal.lua

```diff
@@ -44,7 +44,12 @@ local function is_in_template_tag()
     local has_template_string = false
 
     local current_node = cursor_node
+    local visited_nodes = {}
     while not (has_element and has_template_string) and current_node do
+        if visited_nodes[node_id] then
```

Comment:
Probably forgot to get the current node id 😄 
```suggestion
        local node_id = current_node:id()
        if visited_nodes[node_id] then
```

Reply:
good catch ! 😅